README.md

```diff
@@ -38,15 +38,23 @@ Also, you need to run local node with correct version of code (or change package
 ```bash
 ./ggxchain-node --dev
 
-npm run generate:defs
-npm run generate:types
+bun run generate:defs
```

Comment:
@AuroraLantean Is there a reason to use bun here?

Reply:
@uchu Currently all development linting, formatting, unit testing, and manual testing are using Bun to run. So I thought it is better to gradually move everything to Bun